### PR TITLE
Javadoc Warnings in the Gradle plugin

### LINKED ISSUE #29309: Javadoc Warnings in the Gradle plugin
### Description

While building Quarkus (using `./mvnw -B clean install -DskipTests -DskipITs -DskipDocs -Prelocations`), I get the following warnings while building the Gradle plugin: 

```bash
[INFO] --- exec-maven-plugin:3.0.0:exec (gradle) @ io.quarkus.gradle.plugin ---
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 
> Task :gradle-application-plugin:clean UP-TO-DATE
> Task :gradle-model:compileJava UP-TO-DATE
> Task :gradle-application-plugin:compileJava
> Task :gradle-application-plugin:pluginDescriptors
> Task :gradle-application-plugin:processResources
> Task :gradle-application-plugin:classes
> Task :gradle-application-plugin:jar
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java:141: warning: no @param for fileCollection
    public static File getLastFile(FileCollection fileCollection) {
> Task :gradle-application-plugin:javadoc
                       ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java:141: warning: no @return
    public static File getLastFile(FileCollection fileCollection) {
                       ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:131: warning: no @return
    public Configuration getQuarkusDevConfiguration() {
                         ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:141: warning: no @return
    public FileCollection getSources() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:151: warning: no @return
    public FileCollection getCompilationOutput() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:162: warning: no @return
    public Property<File> getWorkingDirectory() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:170: warning: no @param for workingDir
    public void setWorkingDir(String workingDir) {
                ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:185: warning: no @return
    public boolean isPreventnoverify() {
                   ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:197: warning: no @param for preventNoVerify
    public void setPreventnoverify(boolean preventNoVerify) {
                ^
9 warnings
```

### Implementation ideas

Fix the Javadoc warnings

### PR DESCRIPTION
**Pull request by abhishekS14**
- Fixes #29309

### CODE REVIEW

**Review by gastaldi — DISMISSED** [review]

**Comment by abhishekS14** [comment]
@gastaldi Hi, squashed a change, seems to be some formatting issue during the build

**Comment by gastaldi** [comment]
@abhishekS14 you need to run `mvn process-sources` to have the format applied properly to your changes

**Comment by abhishekS14** [comment]
@gastaldi I have made the changes in the squashed PR, checked locally using the mvn process-sources, does not reflect any new changes after the squashed pr
Should pass the checks

File: .gitattributes [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# ./mvnw fails on Windows in git bash if -Xmx in that file has a trailing CRLF
```
Comment (glefloch) [inline .gitattributes:1]: Could you revert this file? It looks like EOL changed.

File: independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-codestart/java/runtime/src/main/codestarts/quarkus/{extension.id}-codestart/base/src/main/resources/META-INF/resources/index.entry.qute.html [hunk 2]
```diff
@@ -1 +0,0 @@
-{#include index-entry /}
```
Comment (glefloch) [inline independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-codestart/java/runtime/src/main/codestarts/quarkus/{extension.id}-codestart/base/src/main/resources/META-INF/resources/index.entry.qute.html:1]: Is it normal that this file is deleted?

**Review by glefloch — CHANGES_REQUESTED** [review]
Thanks for this work @abhishekS14. I just left two minor comments.

**Comment by abhishekS14** [comment]
I think I will close this, and create a new PR.

**Pull request closed without merging**